### PR TITLE
Upgrade to HTCondor 5.x and fix accounting

### PR DESCRIPTION
**Pull request by timeu**
Modify the accounting script to create APEL log files
for slurm and htcondor-ce.
Remove various workarounds regarding local SLURM submission.

### CODE REVIEW

File: roles/grid/tasks/fetch-crl.yml [hunk 1]
```diff
@@ -15,13 +15,28 @@
...
+    service:
+      name: fetch-crl.service
+      enabled: true
+
+  - name: Enable fetch-crl systemd cron job (Centos8/RHEL8)
+    service:
+      name: fetch-crl.timer
+      enabled: true
+      state: started
+  when: ansible_distribution_major_version == 7
```
Comment (ebirn) [inline roles/grid/tasks/fetch-crl.yml]: should this be 8?

File: roles/htcondor_ce/molecule/default/molecule.yml [hunk 2]
```diff
@@ -9,8 +9,8 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - name: instance-${INSTANCE_ID:-local}
-    hostname: ce.grid.vbc.ac.at
+  - name: instance-centos7-${INSTANCE_ID:-local}
+    hostname: ce-7.grid.vbc.ac.at
```
Comment (ebirn) [inline roles/htcondor_ce/molecule/default/molecule.yml:13]: this one almost got me ;)

**Review by ebirn — CHANGES_REQUESTED** [review]

File: roles/grid/tasks/fetch-crl.yml (hunk 1 above)
Comment (timeu) [reply roles/grid/tasks/fetch-crl.yml]: yes good catch
RHEL8 does not work because there are no repos for it in wlcg and UMD. But I will fix this

**Review by ebirn — APPROVED** [review]
lgtm

**Pull request merged**